src/rules/__tests__/valid-describe-callback.test.ts

```diff
@@ -20,6 +20,7 @@ ruleTester.run('valid-describe-callback', rule, {
     'fdescribe("foo", () => {})',
     'describe.only("foo", () => {})',
     'describe.skip("foo", () => {})',
+    '([]).skip("foo", () => {})',
```

Comment:
seems separate from this pr?

Reply:
it's required for coverage, but I can cover it in the `utils` tests (I'd actually done the refactoring to `findFirstCallPropertyName` a while ago, so forgot it also added this 😅)